arangod/Pregel/Graph.h

```diff
@@ -126,32 +115,21 @@ class Vertex {
   // so it must not allocate any memory or take ownership
   // of anything
 
-  Edge<E>* getEdges() const noexcept { return _edges; }
+  std::vector<Edge<E>> const& getEdges() const noexcept { return _edges; }
 
   // adds an edge for the vertex. returns the number of edges
   // after the addition. note that the caller must make sure that
   // we don't end up with more than 4GB edges per verte.
-  size_t addEdge(Edge<E>* edge) noexcept {
+  size_t emplaceEdge(Edge<E>&& edge) noexcept {
```

Comment:
For me emplace sounds more like a implementation detail - why not use addEdge but still emplace it? add is the name you would expect when you search for such a functionality, but this is perhaps preference?